events/custom.go

```diff
@@ -43,10 +54,17 @@ func (ps *Custom) Add(key string, value interface{}) error {
 	}
 }
 
-func (ps Custom) Len() int           { return len(ps) }
+// Len is a component of the sort.Interface.
+func (ps Custom) Len() int { return len(ps) }
+
+// Less is a component of the sort.Interface.
 func (ps Custom) Less(i, j int) bool { return ps[i].Name < ps[j].Name }
```

Comment:
Why the names? Wouldn't val make more sense?

Reply:
you want a stable sort order by statistics so that "mean_1" is always after "mean_0" snapshot after snapshot, so that it works the right way with ftdc.

Reply:
ack